Dockerfile

```diff
@@ -1,21 +1,50 @@
 FROM r-base
 
 RUN apt-get update \
-    && apt-get install -y wget unzip tar python python3-pip python2 sed bedtools libcurl4-gnutls-dev libxml2-dev libssl-dev gawk \
+    && apt-get install -y wget unzip tar python2 python3 python3-pip  sed bedtools libcurl4-gnutls-dev libxml2-dev libssl-dev gawk \
     && rm -rf /var/lib/apt/lists/*
+RUN wget https://bootstrap.pypa.io/pip/2.7/get-pip.py \
+    && python2 get-pip.py \
+    && rm get-pip.py
 RUN mkdir /gwas-tools
 ENV PATH="/gwas-tools:${PATH}"
 WORKDIR /gwas-tools
-RUN R -e "install.packages(c('mvtnorm', 'corpcor', 'tidyverse', 'magrittr', 'BiocManager'), repos = 'http://cran.us.r-project.org')"
-RUN R -e "BiocManager::install(c('martini','BioNet'))"
+RUN R -e "install.packages(c('mvtnorm', 'corpcor', 'tidyverse', 'magrittr', 'BiocManager', 'cowplot'), repos = 'http://cran.us.r-project.org')"
+RUN R -e "BiocManager::install(c('martini','BioNet','twilight'))"
+#RUN wget http://www.well.ox.ac.uk/~cfreeman/software/gwas/gtool_v0.7.5_x86_64.tgz \
+#    && tar -zxvf gtool_v0.7.5_x86_64.tgz
+#RUN wget https://mathgen.stats.ox.ac.uk/impute/impute_v2.3.2_x86_64_static.tgz \
+#    && tar -zxvf impute_v2.3.2_x86_64_static.tgz && mv impute_v2.3.2_x86_64_static/impute2 .
+#RUN wget http://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64/liftOver && chmod a+x liftOver
 RUN wget https://github.com/bedops/bedops/releases/download/v2.4.35/bedops_linux_x86_64-v2.4.35.tar.bz2 \
     && tar jxvf bedops_linux_x86_64-v2.4.35.tar.bz2 \
     && cp bin/* .
+RUN wget https://github.com/YuanlongLiu/SigMod/raw/master/SigMod_v2.zip \
```

Comment:
We should get some clarification on SigMod's license (if we cannot find a license, we'll have to assume the authors retains all the rights...). We can even inquire them.

An alternative is to use martini's implementation of SigMod, although it's a bit different.

Reply:
The recent update of the repo added an MIT license so I think it is fine

Reply:
Great! Then we just need to fix the installation. It should be pretty easy with devtools.

Reply:
The latest commit still has a build problem. So changes this line in a new commit to take the zip from the [last working commit with our code](https://github.com/YuanlongLiu/SigMod/blob/20c561876d87a0faca632a6b93882fcffd719b17/SigMod_v2.zip)